src/azure-cli-core/azure/cli/core/tooling.py

```diff
@@ -0,0 +1,199 @@
+"""
+Tooling Integration
+
+This module is used by the Azure CLI Tools extension for VSCode. It should not be used
+anywhere else and will be removed in a future release.
```

Comment:
My understanding is that there will be something (like a shell-core module) replacing this in the future and we agreed on this interim solution. @troydai Has the details on the CLI side.